docs/09-references/00-references.adoc

```diff
@@ -92,9 +92,10 @@
 // R
 - [[[richardsfundamentals,Richards+20]]] Mark Richards, Neal Ford: Fundamentals of Software Architecture - An Engineering Approach. OReilly 2020.
 - [[[rozanskiwoods, Rozanski+11]]] Nick Rozanski, Eoin Woods: Software Systems Architecture: Working With Stakeholders Using Viewpoints and Perspectives. Addison-Wesley, 2nd edition 2011.
+
 // S
-- [[[sperberklaeren, Sperber+Klaeren]]] Michael Sperber, Herbert
-  Klaeren: Schreibe Dein Programm!  Tübingen University Press, 2023.  <https://www.deinprogramm.de/sdp/>.
+- [[[solid, SOLID]]] Samuel Oloruntoba and Anish Singh Walia: SOLID: The First 5 Principles of Object Oriented Design, <https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design>.
```

Comment:
Interessante Quelle dafür. war die vorher auch drin?

Reply:
no, we had no reference for SOLID. I wanted to avoid quoting that sexist and reactionist "Uncle Weirdo".